docs/en/latest/plugins/ua-restriction.md

```diff
@@ -2,10 +2,9 @@
 title: ua-restriction
 keywords:
   - APISIX
-  - Plugin
+  - API Gateway
   - UA restriction
-  - ua-restriction
-description: This document contains information about the Apache APISIX ua-restriction Plugin.
+description: This document contains information about the Apache APISIX ua-restriction Plugin, which allows you to restrict access to a Route or Service based on the User-Agent header with an allowlist and a denylist.
```

Comment:
As a user, we may think about how to do xxx with APISIX or API Gateway so that it would be better to provide more cases than a unified style statement in the description field. This is a typical improvement in all documents, and we could do them later.